receiver/hostmetricsreceiver/hostmetrics_receiver_test.go

```diff
@@ -62,7 +62,8 @@ var standardMetrics = []string{
 
 var resourceMetrics = []string{
 	"process.cpu.time",
-	"process.memory.usage",
+	"process.memory.physical_usage",
+	"process.memory.virtual_usage",
```

Comment:
A possible alternative consistent with our other metrics could be:

- process.memory.usage
- process.swap.usage

But I'm not sure about using swap here. This would probably be confusing to people.

Also FYI @aabmass

Reply:
I think it's better to call it virtual. For me at least I associate swap with a swap file (ie disk-backed memory).

Could consider process.memory.physical.usage vs process.memory.physical_usage but I don't feel that strongly one way or another. I can't think of another metrics that might go in that namespace other than process.memory.physical.utilization (for a percentage metric).

Reply:
100% agree

As you said I don't really see physical memory as a category that I would other metrics under so opted for `_` instead of `.`. In any case, we can always change these in the future when a spec is written to confirm the conventions